field/generator/internal/templates/element/fixed_exp.go

```diff
@@ -15,11 +16,11 @@ const FixedExp = `
 {{- end}}
 
 {{define "expByAddChain name data eName"}}
```

Comment:
if we need all these functions public now, maybe we should add a 4th parameter to this template generation function with some extra doc so that the doc is self explanatory